README.md

```diff
@@ -55,27 +59,40 @@ And here are links of other **HugeGraph** component/repositories:
 3. [hugegraph-commons](https://github.com/apache/incubator-hugegraph-commons) (**common & rpc** module)
 4. [hugegraph-website](https://github.com/apache/incubator-hugegraph-doc) (**doc & website** code)
 
-## Contributing
+
+## 🤝 Contributing
 
 Welcome to contribute to HugeGraph, please see [`How to Contribute`](CONTRIBUTING.md) for more information.
```

Comment:
also could add the suggestion below:

`Note: It's recommended to use [GitHub Desktop](https://desktop.github.com/) to greatly simplify the PR and commit process.`

<img width="635" alt="image" src="https://github.com/apache/incubator-hugegraph/assets/17706099/8b39c65a-2d97-4d74-9984-04c9ddfa2bc3">

Reply:
OK, I will add this suggestion.